xtext-gradle-plugin/src/main/java/org/xtext/gradle/tasks/XtextExtension.xtend

```diff
@@ -73,14 +77,14 @@ class XtextExtension {
 
 @Accessors
 class Language implements Named {
-	val String name
```

Comment:
@oehme  the javadoc of the annotation states

> This annotation should be attached to the getter method in Java or the property in Groovy. Annotations on setters or just the field in Java are ignored.

but this does not seem to be true

Reply:
Interesting, I'd have to ask whether this was enhanced at some point.